cs/index.json

```diff
@@ -110,7 +110,18 @@
     "error_try_again" : "Zkusit znovu",
     "error_get_info": "Při získávání těžebních informací se vyskytla chyba",
     "error_hash_verify_failed" : "Při stahování těžebního nástroje se vyskytla chyba, zkuste to znovu",
-    "error_unzip" : "Při rozbalování souboru zip se vystytla chyba, zkuste to znovu"
+    "error_unzip" : "Při rozbalování souboru zip se vystytla chyba, zkuste to znovu",
+    "initializing": "Checking mining tool information",
```

Comment:
I don't think the key is a good one since it has nothing to do with the message itself?

Reply:
I think it has meaning because of this key put inside miner_download object

Reply:
I think checking_info would make more sense compared to all the other keys?

Reply:
Okay, But please keep this key for now, I forgot check this PR but merge main MR in https://gitlab.com/mcnfreedom/vn-team/crypto.miner.client/merge_requests/176. I will note it and update later